companion/src/modeledit/setup.cpp

```diff
@@ -42,6 +42,13 @@ TimerPanel::TimerPanel(QWidget *parent, ModelData & model, TimerData & timer, Ge
     ui->countdownBeep->addItem(tr("Voice"), TimerData::COUNTDOWN_VOICE);
     ui->countdownBeep->addItem(tr("Haptic"), TimerData::COUNTDOWN_HAPTIC);
   }
+  
+  if (IS_TARANIS(board))
+    ui->value->setMaximumTime(QTime(23, 59, 59));
+  else if (IS_ARM(board))
+    ui->value->setMaximumTime(QTime(8, 59, 59));
+  else
+    ui->value->setMaximumTime(QTime(0, 59, 59));
```

Comment:
This and the same section above should go into the `firmware->getCapability()` function.